docs/src/content/docs/guides/CLI-API/migrating_rdm_disk_windows_cluster_machine_using_cli.md

```diff
@@ -113,29 +179,37 @@ status:
   phase: Available
 ```
 
-### 7. Ensure all VM's of cluster is migrated
+### 7. Ensure All the VMs in Cluster are Migrated
+
+1. Check that the RDM disk is available as a volume in PCD or OpenStack.
+
+2. Ensure all VMs in the cluster are migrated.
+
+3. Power on all VMs together.
 
-1) Check RDM disk is available as volume in PCD or openstack
+### Rollback Plan - If Migration Fails
 
-2) Ensure all VM's of cluster is migrated
+1. Delete VMs created in PCD or OpenStack.
+2. Delete the managed volume:
 
-3) Power on all VM's together
+   ```bash
+   openstack volume delete <volume-id> --remote
+   ```
 
-### Rollback plan - if migration fails
+  **For cli client versions less than 2025.1:**
+- [Volume Delete](https://docs.redhat.com/en/documentation/red_hat_openstack_platform/10/html/command-line_interface_reference_guide/openstackclient_subcommand_volume_delete)
 
-1) Delete VMs created in PCD or openstack
-2) Delete managed volume 
-``` openstack volume delete volumeid --remote```
-3) Attach RDM disk in VMware to powered off VM's
+**For cli client versions >= 2025.1:**
+- [Cinder Command Options Reference](https://docs.openstack.org/python-openstackclient/latest/contributor/command-options.html)
```

Comment:
I think we can remove this, we have already mentioned that minimum CLI version is 6.2.1 which will have this command we need.